task/monorail-server/src/main.rs

```diff
@@ -5,6 +5,11 @@
 #![no_std]
 #![no_main]
 
+#[cfg_attr(target_board = "sidecar-a", path = "bsp/sidecar_1.rs")]
```

Comment:
Add a `#[deny(unused_attributes)]`

Reply:
I ended up doing that globally in `build/dist.rs`